### PR TITLE
Adding conf and suites for quincy

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Adding Conf and suites for Quincy
This exercise is to maintain suites and confs for each releases seperately 

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/quincy/cephfs/tier-1_cephfs_mirror.yaml [hunk 1]
```diff
@@ -0,0 +1,254 @@
+#===============================================================================================
```
Comment (sumabai) [inline suites/quincy/cephfs/tier-1_cephfs_mirror.yaml:1]: please check if tier-2_cephfs_test-nfs_ha.yaml needs to be added

File: suites/quincy/cephfs/tier-1_cephfs_mirror.yaml [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+# Tier-level: 1
+# Test-Suite: tier-1_cephfs_mirror.yaml
+# Test-Case: Configure CephFS Mirror setup and run IOs
```
Comment (sumabai) [inline suites/quincy/cephfs/tier-1_cephfs_mirror.yaml:4]: Do we also need mention about relevant/suggested conf file for the test suites in documentation?

File: suites/quincy/cephfs/tier-1_fs.yaml [hunk 3]
```diff
@@ -0,0 +1,217 @@
+tests:
```
Comment (sumabai) [inline suites/quincy/cephfs/tier-1_fs.yaml:1]: some documentation for test suite description and suggested conf file?

File: suites/quincy/cephfs/tier-1_cephfs_mirror.yaml (hunk 1 above)
Comment (neha-gangadhar) [reply suites/quincy/cephfs/tier-1_cephfs_mirror.yaml:1]: +1 the suites is missing in quincy folder.

**Comment by neha-gangadhar** [comment]
Can you please check and include 2 more suites : 
tier-2_cephfs_test-snapshot-clone.yaml
tier-3_cephfs_scale.yaml

**Comment by AmarnatReddy** [comment]
> Can you please check and include 2 more suites : tier-2_cephfs_test-snapshot-clone.yaml tier-3_cephfs_scale.yaml

These are already to included

**Review by neha-gangadhar — APPROVED** [review]
lgtm

**Pull request merged**